doc/src/user_guide.rst

```diff
@@ -129,6 +129,40 @@ use the following settings in the configuration file (``.ini``)::
 Note the version of the compiler which must support the ``openmp`` flag.
 This has been tested on Mac OS 10.14.
 
+Quick-start Ubuntu 18
+^^^^^^^^^^^^^^^^^^^^^
+
+Open the terminal and install the dependencies with the following commands::
+
+    sudo apt install python3 python3-pip libopenmpi-dev openmpi-bin metis libmetis-dev libblas3
+    pip3 install virtualenv
+
+For visualisation of results, either install Paraview from the command line::
```

Comment:
Why do we have lots of text about Paraview here, when in the OSX guide we just have a single brew install command? Can you make the two guides consistent?

Reply:
sure